docs/content/topics/project-developers/events.md

```diff
@@ -9,4 +9,229 @@ aliases:
   - /topics/project-developers/events.md
 ---
 
-TODO
\ No newline at end of file
+# Brigade Events
+
+Events are the lingua franca in Brigade: [Gateways] emit events and [Project]
```

Comment:
Total nit, but "lingua franca" should probably be italicized, as that's the common convention for foreign words.

This is so much of a nit that I don't care at all if you ignore it.